### DISCUSSION TITLE
How to listen for hover on edges

### DISCUSSION THREAD

**Discussion by slechanii** [xyflow/xyflow#1940] [Q&A] [answered]
Hi, I'm looking for a way to do something when the user hovers on an edge (I want to change the edge type in my store to display things differently when a user hovers on an edge)

I haven't found the correct way to listen for hovers on links yet, does someone know how I could achieve this please ?

Thanks

**Comment by moklick** [comment, marked as answer]
Hey @slechanii 

`onEdgeMouseEnter` and `onEdgeMouseLeave` should do what you want.

**Reply by slechanii** [reply]
Thanks a lot for the quick answer!
